src/api/app/models/staging/staged_requests.rb

```diff
@@ -84,6 +85,34 @@ def requests
     staging_workflow.unassigned_requests.where(number: request_numbers)
   end
 
+  def link_package(bs_request_action)
+    request = bs_request_action.bs_request
+
+    source_package = Package.get_by_project_and_name!(bs_request_action.source_project,
+                                                      bs_request_action.source_package)
+
+    # get the base revision from the source package
+    baserev = source_package.backend_package.srcmd5
+
+    # create the package in the staging project
+    link_package = Package.create!(project: staging_project,
+                                   name: bs_request_action.source_package)
+
+    # add link file to the project, linking to the source project baserev
+    link_package.save_file(filename: '_link', file: "<link project=\"#{source_package.project.name}\" baserev=\"#{baserev}\"></link>")
```

Comment:
what does the baserev to do with this?

Reply:
I was just mimicking what I saw in the `_link` file and it worked, package can be built. Any suggestion?

Reply:
In what _link file btw?